app/views/partials/account_summary/vat/account_summary.scala.html

```diff
@@ -23,74 +23,75 @@
+<section>
+  <h2 style="margin-top: 1.875em !important">@messages("vat.your_payments")</h2>
+    <p>
+      <span id="vat-balance">
+      @Html(balanceInformation)
+      </span>
+    </p>
+    <p>
+         @breakdownLink.map { link =>
+      <a id="vat-see-breakdown-link" href="@link"
+        data-journey-click="link - click:VATaccountSummary:how we worked this out/view statement"
+        target="_blank" rel="external noopener">@breakdownText</a>
         }
-        @fileReturnLink
-    </li>
-    <li>
-        @if(panelIndent) {
-        <div class="panel-indent">
-        }
-        <p>
-            <span id="vat-balance">
-            @Html(balanceInformation)
-            </span>
-               @breakdownLink.map { link =>
-            -
-            <a id="vat-see-breakdown-link" href="@link"
-            data-journey-click="HomepageVAT:click:SeeBreakdown">@breakdownText</a>
-        }
-        </p>
-        @if(panelIndent) {
-        </div>
-        }
-
-        @if(showRepaymentContent) {
-            <details id="vat-repaid-details">
-                <summary class="underline">
-                    <span id="vat-when-repaid">@Messages("vat.when.repaid")</span>
-                </summary>
-                <div class="panel-indent">
-                    <p>@Html(messages("vat.when.repaid.detail.1", "<a id=\"vat-repayments-account\" " +
-                            "data-journey-click=\"VAT:click:RepaymentsBankAccount\" href=\"" +
-                            appConfig.getBusinessAccountUrl("manageAccount") + "#bank\">", "</a>"))</p>
-                    <p>@Messages("vat.when.repaid.detail.2")</p>
-                    <p>@Html(messages("vat.when.repaid.detail.3", "<a id=\"vat-more-than-21-days\" " +
-                            "data-journey-click=\"VAT:click:MoreThan21Days\" href=\"" +
-                            appConfig.getGovUrl("vatEnquiries") + "\">", "</a>"))</p>
-                </div>
-            </details>
-        }
-    </li>
-    <li class="section__li">
-        <a id="vat-make-payment-link"
-        href="@appConfig.getUrl("makeAPayment")"
-        class="button button--payment"
-        data-journey-click="vat:Click:Make a VAT payment">
-        @Html(messages("vat.link.message.accountSummary.makeAPayment"))
-        </a>
-    </li>
-
-    @directDebitContent
-
-</ul>
+    </p>
 
+  @if(showRepaymentContent) {
+      <details id="vat-repaid-details">
+          <summary class="summary--repaid underline">
+              <span id="vat-when-repaid">@Messages("vat.when.repaid")</span>
+          </summary>
+          <div class="panel-indent">
+              <p>
+                @messages("vat.we_will_transfer_this_amount_")
+                <a id="vat-repayments-account"
+                  data-journey-click="link - click:VATaccountSummary:repayments bank account"
```

Comment:
Is this a valid GA tracking code?

Reply:
It's my understanding from working with Adam that this is the style we want going forward - I recommend double-checking with him.